### PR TITLE
latLngBounds helper

### PR DESCRIPTION
**Pull request by lucacorti**
What about an helper for bounds too?

### CODE REVIEW

**Comment by miguelcobain** [comment]
@lucacorti can you please provide an example of where we could use this?

**Comment by lucacorti** [comment]
The use case is setting the bounds property in map initialization

> {{#leaflet-map bounds=(latLngBounds ...) ...}}

**Comment by miguelcobain** [comment]
Mind adding a test?
Example: https://github.com/miguelcobain/ember-leaflet/blob/master/tests%2Funit%2Fhelpers%2Fdiv-icon-test.js

**Comment by miguelcobain** [comment]
I also make sure helpers are compatible with tests like these: https://github.com/miguelcobain/ember-leaflet/blob/master/tests%2Fintegration%2Fcomponents%2Fmarker-layer-test.js#L158-L185

**Comment by lucacorti** [comment]
The leaflet docs state that simple array form is accepted in place of bounds. I tested this passing an array of two element arrays (points) to the leaflet component bounds property.

http://leafletjs.com/reference.html#bounds

and should work for points too

http://leafletjs.com/reference.html#point

unless there actually are exceptions tho this.

**Comment by miguelcobain** [comment]
What I meant is that I'm ok to merge this in if you add a test for this.

**Comment by miguelcobain** [comment]
@lucacorti ping

**Comment by lucacorti** [comment]
Sorry, I'm having some trouble running the tests under node 6.0. I'll let you know as soon as I get that working.

**Comment by lucacorti** [comment]
Added a test and fixed code, of course 😄 
`ember-cli-sass` needed updating to work on node >= 6.0
Failing tests on canary/beta seem unrelated.

File: tests/unit/helpers/lat-lng-bounds-test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { latLngBounds } from 'dummy/helpers/lat-lng-bounds';
+import { module, test } from 'qunit';
+/* global L */
+
+module('Unit | Helper | lat-lng-bounds');
+
+// Replace this with your real tests.
```
Comment (miguelcobain) [inline tests/unit/helpers/lat-lng-bounds-test.js]: Can you just remove this line? :)

Comment (miguelcobain) [inline tests/unit/helpers/lat-lng-bounds-test.js]: On the other helpers as well.

**Comment by miguelcobain** [comment]
@lucacorti great job here!

Just onde teste missing. Let's teste that the helper works with leaflet-map. Let's show people this helper in action.

I did something similar to the resto of the helpers: https://github.com/miguelcobain/ember-leaflet/blob/master/tests%2Fintegration%2Fcomponents%2Fmarker-layer-test.js#L158-L185

**Comment by miguelcobain** [comment]
@lucacorti we also need to add a page for this in the docs, in the "helpers" section.

**Comment by ivanhueso** [comment]
Hi @miguelcobain , What is the status of adding the bounds, is something i can start using? Thank you Ivan

**Comment by lucacorti** [comment]
I have added a test and an helper section in the docs.

File: tests/integration/components/leaflet-map-test.js [hunk 2]
```diff
@@ -145,3 +146,17 @@ test('setting zoom to 0 should not throw', function(assert) {
...
+  this.set('markerCenter', locations.nyc);
+  this.set('bounds', locations.bounds());
+
+  this.render(hbs`
+    {{#leaflet-map bounds=(lat-lng-bounds bounds)}}
+      {{marker-layer location=markerCenter}}
+    {{/leaflet-map}}
+  `);
+
+  assert.ok(map._layer.getBounds() instanceof L.LatLngBounds);
```
Comment (miguelcobain) [inline tests/integration/components/leaflet-map-test.js]: Here you're just testing that `getBounds` returns an instance of `L.LatLngBounds`. That test isn't meaningful.
We want to test that the bounds are what we expect them to be.

Please use the `boundsContain` helper, like here: https://github.com/miguelcobain/ember-leaflet/blob/master/tests%2Fintegration%2Fcomponents%2Fleaflet-map-test.js#L78

**Comment by ivanhueso** [comment]
Thanks @lucacorti it works!

**Comment by miguelcobain** [comment]
@lucacorti many thanks for your contribution!
`2.2.6` will be released soon.

**Pull request merged**